package_policies/R10.md

```diff
@@ -0,0 +1,56 @@
+**R10.** Each package should provide pre-processor macros that allow for version
+comparison (for languages that support it - like C/C++/Fortran).
+Using these macros, dependent software can potentially be compatible
+with multiple versions of the package.
+
+Example implementations for a given `package version 1.2.3`
+
+- Example 1: version macros, can be done easily with CMake
```

Comment:
@mkstoyanov 

Perhaps this is what you want to convey

```suggestion
- Example 1: version macros only , via public include files [or can be done easily with CMake]
```

Also since there is emphasis on making the (Example)  comparison macros optional - and current organization 1,2,3 is not conveying that - I'll reorganize them a bit.

Reply:
@mkstoyanov @v-dobrev @shuds13 I've reorganized the examples a bit - please recheck.